### PR TITLE
Merging changes into main

### PR DESCRIPTION
**Pull request by AlbertoCasasOrtiz**
Fixes:

- #231 
- #370 
- #374 
- #376 
- #378

### CODE REVIEW

**Comment by AlbertoCasasOrtiz** [comment]
@carmichaelong This PR introduces some changes into prod. Most of them are minor bugs in the GUI, but https://github.com/stanfordnmbl/opencap-viewer/issues/231 touches a lot of pats of the code. I will later today test a full pass through the app, but could you test as well to check i didn't miss anything? Everything should be in dev already.

**Comment by carmichaelong** [comment]
- Whn you click record in step 4, there is no feedback #370
The record button goes away, but another message noting that there are no phones connected and they should go back to another step would be helpful.

- Attempting to rename a trial throws an error #374
This was fixed in an earlier dev version but seems to have come back. Perhaps that code just was just accidentally left out of this PR?

- Sessions not shown when subject is selected in subjects view #376
This looks good to me.

- Add a message in the link to phone page so users know the opencap app is now in AppStore #378
This looks good to me.

**Comment by AlbertoCasasOrtiz** [comment]
> * [Whn you click record in step 4, there is no feedback #370](https://github.com/stanfordnmbl/opencap-viewer/issues/370)
>   The record button goes away, but another message noting that there are no phones connected and they should go back to another step would be helpful.

I added a message indicating to go back to the previous step in case there are less than 2 phones connected. Regarding the disappearing button, I think it is not a bug as it disappears when Record is clicked and the neutral pose is being processed. Once finished it takes you automatically to the next step, or reappears if there has been an error so you can Record again.

> * [Attempting to rename a trial throws an error #374](https://github.com/stanfordnmbl/opencap-viewer/issues/374)
>   This was fixed in an earlier dev version but seems to have come back. Perhaps that code just was just accidentally left out of this PR?

You are right, when merging this was left out. I re-added it.

File: src/components/pages/Neutral.vue [hunk 1]
```diff
@@ -688,74 +688,82 @@ export default {
...
-              }
-            );
-
-            const resSubject = await axios.get(
-                `/sessions/${this.session.id}/set_subject/`,
+        if (this.n_calibrated_cameras < 2) {
+          if (this.n_calibrated_cameras < 1)
+              apiError("No cameras have been calibrated. Please go back and calibrate your cameras.");
+          else if (this.n_calibrated_cameras == 1)
+              apiError("There is only 1 calibrated camera, but >2 are necessary. Please go back and calibrate your cameras.");
```
Comment (carmichaelong) [inline src/components/pages/Neutral.vue:695]: consider replacing `>2` with `at least 2 cameras`

**Comment by carmichaelong** [comment]
Both of those changes look good to me. Just made one minor comment on changing the wording of an error message.

**Comment by antoinefalisse** [comment]
@AlbertoCasasOrtiz and @carmichaelong anything stopping this one from being merged?

**Comment by AlbertoCasasOrtiz** [comment]
Merging as everything has been tested in dev,

**Pull request merged**